apps/docs/src/app/core/component-docs/list/examples/list-nav-indicator-example/list-nav-indicator-example.component.html

```diff
@@ -0,0 +1,30 @@
+<ul fd-list [hasNavigation]="true" [navigationIndication]="true">
+    <li fd-list-item [link]="true">
```

Comment:
How I can set my target

Reply:
you can just use routerLink or attr.href on the anchor element

Reply:
you can just use routerLink or attr.href on the anchor element